www/calendar/inner.js

```diff
@@ -845,12 +861,13 @@ define([
                     return t1 > t2 ? 1 : (t1 === t2 ? 0 : -1);
                 });
             };
+
             var tempCalendars = filter(0);
             if (tempCalendars.length && tempCalendars[0] === APP.currentCalendar) {
-                APP.$calendars.append(h('div.cp-calendar-team', [
+                var $tempCalendarTeam = $(h('div.cp-calendar-team', [
                     h('span', Messages.calendar_tempCalendar)
-                ]));
-                makeCalendarEntry(tempCalendars[0], 0);
+                ])).appendTo(APP.$calendars);
+                var $tempCalendarEntries = appendCalendarEntries(0, filter).appendTo(APP.$calendars);
```

Comment:
I don't think "appendCalendarEntries" is a good name for this function. As far as I understand, this function creates a container for the calendar entries and fills it, but the name suggests that it is going to append these entries to an existing element.